test/e2e/common/container_probe.go

```diff
@@ -414,9 +415,12 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		framework.ExpectNoError(err)
 
 		framework.Logf("Container started at %v, pod became ready at %v", startedTime, readyTime)
-		if readyTime.Sub(startedTime) < 40*time.Second {
+		if readyTime.Sub(startedTime) < sleepBeforeStarted*time.Second {
 			framework.Failf("Pod became ready before startupProbe succeeded")
 		}
+		if readyTime.Sub(startedTime) > (sleepBeforeStarted+20)*time.Second {
```

Comment:
readiness probe is configured with the default period which is 10 seconds. 

> `periodSeconds`: How often (in seconds) to perform the probe. Default to `10` seconds. Minimum value is `1`.

so this check doesn't validate that it was triggered by the manual trigger. What am I missing?

Reply:
changed to 60s